src/app/containers/Caption/index.jsx

```diff
@@ -10,36 +11,32 @@ import InlineLink from '../InlineLink';
 
 const componentsToRender = { fragment: Fragment, urlLink: InlineLink };
 
-const renderText = textBlocks => (
-  <Blocks blocks={textBlocks} componentsToRender={componentsToRender} />
+const renderParagraph = paragraphBlock => (
+  <Paragraph key={deepGet([0, 'model', 'text'], paragraphBlock)}>
```

Comment:
We aren't passing `script` here which is required in Paragraph, planning to address this in https://github.com/bbc/simorgh/pull/1692